modules/cluster-logging-removing-unused-components-if-no-elasticsearch.adoc

```diff
@@ -7,7 +7,7 @@
 
 As an administrator, in the rare case that you forward logs to a third-party log store and do not use the default Elasticsearch log store, you can remove several unused components from your logging cluster.
 
-In other words, if you do not use the default Elasticsearch log store, you can remove the internal Elasticsearch `logStore`, Kibana `visualization`, and log `curation` components from the `ClusterLogging` custom resource (CR). Removing these components is optional but saves resources.
+In other words, if you do not use the default Elasticsearch log store, you can remove the internal Elasticsearch `logStore` and Kibana `visualization` components from the `ClusterLogging` custom resource (CR). Removing these components is optional but saves resources.
```

Comment:
`In other words` seems superfluous. Consider rephrasing to: `If you do not use the default Elasticsearch log store, you can remove the...`

Reply:
Thank you, Jake. I'd like to keep "In other words" as a signal to the reader that this phrase repeats and expands on the information in the preceding statement.